external-import/recorded-future/src/config.yml.sample

```diff
@@ -5,29 +5,28 @@ opencti:
 connector:
   id: 'ChangeMe'
   type: 'EXTERNAL_IMPORT'
-  name: 'Recorded Future Analyst Notes' # Required
+  name: 'Recorded Future' # Required
   scope: 'ipv4-addr,ipv6-addr,vulnerability,domain,url,file-sha256,file-md5,file-sha1'
-  confidence_level: 5
   update_existing_data: false
   log_level: 'info'
 
 rf-notes:
```

Comment:
There are a few notes left ! the little scamps xD
- In config.yml.sample (rf-notes)
- In main.py (all name-related get_config_variables in config.yml.sample, for example: ["rf-notes", "token"])